README.md

```diff
@@ -4,3 +4,11 @@
 [![Dependency Status](https://gemnasium.com/badges/github.com/bm-sms/daimon-news.svg)](https://gemnasium.com/github.com/bm-sms/daimon-news)
 
 NOTE: The documentation is in progress.
+
+## Requirements
+
+If you use macOS and have not used groonga:
+
+    $ brew install mecab
+    $ brew install mecab-ipadic
+    $ brew install groonga --with-mecab
```

Comment:
The dependencies for groonga with the `mecab` option are automatically resolved. So you don't have to run `brew install mecab` and `brew install mecab-ipadic` in advance, right? 😻

<img width="570" alt="screen shot 2016-12-20 at 16 58 29" src="https://cloud.githubusercontent.com/assets/7645585/21342500/931f553c-c6d5-11e6-9b69-4b488345566b.png">